src/app/search-navbar/search-navbar.component.ts

```diff
@@ -67,7 +67,7 @@ export class SearchNavbarComponent {
 
     this.router.navigate(linkToNavigateTo, {
       queryParams: queryParams,
-      queryParamsHandling: 'merge'
+      queryParamsHandling: ''
```

Comment:
Currently, if you have a search page open with a number of filters (e.g. https://sandbox.dspace.org/search?spc.page=1&f.author=Simmons,%20Cameron,equals ) and you enter a query from the header, you won't lose those filters.

Removing 'merge' here will ensure that you will lose any preexisting filters. I'm not sure whether or not we want this, but it should be a conscious decision.

I'm also not sure why the change was needed for this PR